### PR TITLE
Sample react app deployed in jaggeryapps + server-side routing -Store

### PR DESCRIPTION
**Pull request by kavishkafernando**
This PR includes a Sample react app deployed in jaggeryapps (Store).
This PR also includes server-side routing for the react-app.
All the files included in the react app folders are sample files and should be replaced.
jaggery.conf and public/jaggery/index.jag should not be replaced.
org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new folder contains all the react resources.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/jaggery.conf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "errorPages":
+         {
+            "401":"/site/pages/error-pages/401.html",
+            "403":"/site/pages/error-pages/403.html",
+            "404":"/site/pages/error-pages/404.html",
+            "500":"/site/pages/error-pages/500.html",
+            "0":"/site/pages/error-pages/error-page.html"
+         }
+    ,
+
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/jaggery.conf]: We can remove these unnecessary new lines.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/public/js/hello.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<body>
+
+<h2>JavaScript in Head</h2>
+
+<p id="demo">A Paragraph.</p>
+
+<button type="button" onclick="myFunction()">Try it</button>
+
+</body>
+</html>
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/public/js/hello.js]: Add newline at the end of the file.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/pages/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <link href="css/StyleSheet.css" rel="stylesheet" type="text/css">
+    <script src="js/hello.js"></script>
+</head>
+<body>
+<h1>Hello</h1>
+<img src="images/download.jpeg" alt="Hello">
+
+</body>
+</html>
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/pages/index.html]: Add a newline.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/public/js/hello.js (hunk 2 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/public/js/hello.js]: This was a sample file, this file is no longer in the repo.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/pages/index.html (hunk 3 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/pages/index.html]: This was a sample file, this file is no longer in the repo.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/jaggery.conf (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/jaggery.conf]: Has been removed with commit [05721a7449c0fdd5ea8bf15b61ce3da047f9bba2](https://github.com/wso2/carbon-apimgt/commit/05721a7449c0fdd5ea8bf15b61ce3da047f9bba2)

**Pull request merged**